src/components/shuttle/AutonomousShuttlePlatoonWidget.tsx

```
@@ -0,0 +1,326 @@
+        </div>
+      )}
+
+      {/* Demand Slider & Platoon Optimizer Controller */}
+      <div className="border-2 border-black bg-white p-4 space-y-4 shadow-[4px_4px_0_0_#000] dark:bg-zinc-800">
+        <div className="flex flex-col sm:flex-row sm:items-center justify-between gap-2">
+          <label className="font-mono text-xs font-black uppercase text-black dark:text-white flex items-center gap-1.5">
+            <Gauge className="h-4 w-4 text-sky-600" />
+            Simulate Event Surge Demand (Waiting Attendees):
+          </label>
+          <span className="font-mono text-xs font-bold text-purple-900 bg-purple-100 border border-black px-2 py-0.5">
+            Recommended Convoy Size: {surgeEval.recommendedPlatoonSize} Units ({surgeEval.recommendedPlatoonSize * 14} Seats)
```

ESLint

[error] Insert `⏎············`

(prettier/prettier)